Agro_easy/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import About from './components/about';
+import Home from './components/home';
+
+
+class App extends Component {
+    render() {
+        return (
+            <Router>
+                <div className="App">
+                   <Route path='/' exact strict component={Home} />
```

Comment:
what does adding `exact strict` do for the Route component here?

Reply:
it ensure a page renders correclly, like /about/